source/js/components/details.js

```diff
@@ -6,29 +6,26 @@ import ReactMarkdown from "react-markdown";
 var items = [
   {
     source: require("../../markdown/details/item-1.md"),
-    header: "How to enter the challenge"
+    header: "Who can enter the challenges"
   }, {
     source: require("../../markdown/details/item-2.md"),
-    header: "Timeline"
```

Comment:
Yeah, but I figured the markdown could be updated by mehan. (the current content is pretty inaccurate as is, I didn't see value in worrying about it.)

It might make sense for me to just clear the content maybe?